### PR TITLE
Add translations: ru.errors.messages.in, ru.number.format.round_mode.

### PR DESCRIPTION
**Pull request by mpestov**
Hello guys!
I would like to add some translations for Russian locale:
```
ru:
  errors: 
    messages: 
      in: должно быть в диапазоне %{count}

  number:
    format:
      round_mode: default
``` 

Changelog actualized.
Thank you in advance.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
   - Japanese (ja): Add `in` and `round_mode` keys #1059
   - Korean (ko): Fix typo in `equal_to` keys #1061
   - Portuguese (pt, pt-BR): add translation for `errors.messages.in` #1071
-  - Russian (ru): fix some errors in 'datetime' section
+  - Russian (ru): fix some errors in 'datetime' section, add `errors.messages.in` and `number.format.round_mode` keys
```
Comment (sunny) [inline CHANGELOG.md:11]:
```suggestion
  - Russian (ru): fix some errors in 'datetime' section, add `errors.messages.in` and `number.format.round_mode` keys #1077
```

**Review by sunny — APPROVED** [review]
Thank you! Sounds good to me 👍🏻

File: CHANGELOG.md (hunk 1 above)
Comment (mpestov) [reply CHANGELOG.md]: I did it)

**Comment by pama** [comment]
Thanks!

**Pull request merged**